scss/index.scss

```diff
@@ -46,4 +46,5 @@ p {
 
 // Local SCSS Imports
 @import './challengeMaterial.scss';
+@import './modalCard.scss';
```

Comment:
`modalCard` is global because it can't access 3rd party component. It modifies `react-bootstrap` styles and not its own.